src/main/java/com/iexec/core/task/TaskStatus.java

```diff
@@ -27,6 +27,7 @@
     INITIALIZED,
     INITIALIZE_FAILED,
     RUNNING,
+    ALL_WORKERS_FAILED,
```

Comment:
RUNNING_FAILED might be more appropriate to keep up with the workflow, it precisely describes an all failed before replicate contributions, whereas ALL_WORKERS_FAILED tends to express it did happen on any other task stage.

Reply:
Fixed!